packages/android_intent/android/src/main/java/dev/flutter/plugins/androidintent/AndroidIntentPlugin.java

```diff
@@ -0,0 +1,66 @@
+package dev.flutter.plugins.androidintent;
+
+import androidx.annotation.NonNull;
+import io.flutter.embedding.engine.plugins.FlutterPlugin;
+import io.flutter.embedding.engine.plugins.activity.ActivityAware;
+import io.flutter.embedding.engine.plugins.activity.ActivityPluginBinding;
+import io.flutter.plugin.common.MethodChannel;
+import io.flutter.plugin.common.PluginRegistry.Registrar;
+
+/**
+ * The new embedding implementation of the plugin.
```

Comment:
I'm not sure if this is temporary or not, but is it a good idea to include temporal references in docs?

Reply:
I tried to future proof it a little.